playbooks/roles/pachyderm/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
    command: >
      helm upgrade --install
      --version "0.1.4"
-     --set pachd.image.tag="{{ pachyderm_tag }}",pachd.worker.tag="{{ pachyderm_tag }}",etcd.image.tag="{{ etcd_tag }}",credentials=s3,s3.accessKey="{{ pachyderm_minio_accesskey }}",s3.secretKey="{{ pachyderm_minio_secretkey }}",s3.bucketName=defaultbucket,s3.endpoint="{{ minio_release_name }}"-minio-svc.default.svc.cluster.local:9000,s3.signature=0,etcd.persistence.enabled=true,etcd.persistence.accessMode=ReadWriteMany,etcd.persistence.size="{{ pachyderm_etcd_pvc_size }}",etcd.resources.requests.cpu=1,etcd.resources.requests.memory=2G,pachd.resources.requests.cpu=1,pachd.resources.requests.memory=2G,pachd.pfsCache=1G
+     --set pachd.image.tag="{{ pachyderm_tag }}",pachd.worker.tag="{{ pachyderm_tag }}",etcd.image.tag="{{ etcd_tag }}",credentials=s3,s3.accessKey="{{ pachyderm_minio_accesskey }}",s3.secretKey="{{ pachyderm_minio_secretkey }}",s3.bucketName=defaultbucket,s3.endpoint="{{ minio_release_name }}"-minio-svc.default.svc.cluster.local:9000,s3.signature=0,etcd.persistence.enabled=true,persistence.storageClass="{{ os_sc }}",etcd.persistence.accessMode=ReadWriteMany,etcd.persistence.size="{{ pachyderm_etcd_pvc_size }}",etcd.resources.requests.cpu=1,etcd.resources.requests.memory=2G,pachd.resources.requests.cpu=1,pachd.resources.requests.memory=2G,pachd.pfsCache=1G
      "{{ pachyderm_release_name }}" stable/pachyderm
 
  - name: wait for etcd to be Running
```

Comment:
Why is it called wait for etcd running?

Reply:
@andersla because `etcd` needs to be running before `pachd`. It's part of the deployment.

Reply:
I just didn't look at the whole code...;)